config/styles/blade-effects.md

```diff
@@ -64,58 +99,33 @@ When using a feature that skips preon.
 ### EFFECT_PLI_ON  
 ### EFFECT_PLI_OFF  
 
-## User-definable effects  
-### EFFECT_USER1  
-### EFFECT_USER2  
-### EFFECT_USER3  
-### EFFECT_USER4  
-
-## ----- The following are as of ProffieOS7 -----
-
-### EFFECT_VOLUME_LEVEL    
-Show volume level visually on blade, great for using with volume menu feature.  
+## Miscelaneous
 
-### EFFECT_QUOTE  
-### EFFECT_SECONDARY_IGNITION  
-### EFFECT_SECONDARY_RETRACTION  
-### EFFECT_OFF  
-
-### EFFECT_FAST_OFF  
-When using a feature that bypasses postoff.  
-
-### EFFECT_OFF_CLASH  
-### EFFECT_NEXT_QUOTE  
-### EFFECT_INTERACTIVE_PREON  
-### EFFECT_INTERACTIVE_BLAST  
-### EFFECT_TRACK  
-### EFFECT_BEGIN_BATTLE_MODE  
-### EFFECT_END_BATTLE_MODE  
-### EFFECT_BEGIN_AUTO_BLAST  
-### EFFECT_END_AUTO_BLAST  
-
-### EFFECT_ALT_SOUND  
-Triggers the change for sets of sounds within the font from one alternative to another.  
-
-### EFFECT_TRANSITION_SOUND  
-Triggers an optional sound effect during transitions from within a style via TrDoEffect.  
+### EFFECT_NONE
```

Comment:
Putting EFFECT_NONE in the middle is weird.
It should either go in the beginning, end or not be a part of this page at all, since it's not really a usable effect.